Assets/MouseLook.cs

```diff
@@ -0,0 +1,29 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class MouseLook : MonoBehaviour
+{
+    public const float MouseSensitivity = 100f;
+
+    public Transform PlayerBody;
+
+    float xRotation = 0f;
```

Comment:
```suggestion
    private float xRotation = 0f;
```
always include the `public` or `private`